pkg/app/server.go

```diff
@@ -376,6 +378,18 @@ func buildTelemetryServer(registry prometheus.Gatherer) *http.ServeMux {
 	// Add metricsPath
 	mux.Handle(metricsPath, promhttp.HandlerFor(registry, promhttp.HandlerOpts{ErrorLog: promLogger{}}))
 
+	// Add readyzPath
+	mux.Handle(readyzPath, http.HandlerFunc(func(w http.ResponseWriter, _ *http.Request) {
+		count, err := testutil.GatherAndCount(registry)
```

Comment:
I like the approach, I am a bit concerned that we're getting a testutil into a production path.
Is this considered stable?

Reply:
"Package testutil provides helpers to test code using the prometheus package of client_golang."

Reply:
I've extracted the function in a manner that makes it only dependent on the official library, PTAL.